src/promptflow-evals/promptflow/evals/evaluate/_eval_run.py

```diff
@@ -249,8 +249,7 @@ def get_metrics_url(self):
     def _get_token(self):
         # We have to use lazy import because promptflow.azure
         # is an optional dependency.
-        from promptflow.azure._utils._token_cache import ArmTokenCache
-        return ArmTokenCache().get_token(self._ml_client._credential)
+        return self._ml_client._credential.get_token(EvalRun._SCOPE)

```

Comment:
Could you please add a TODO so we don't forget it later on?

Reply:
Added to task in ADO to track it.